src/app/components/app-layout/app-layout.component.html

```diff
@@ -37,15 +37,17 @@
     <div *ngIf="isAuthenticated" class="profile">
       <div class="btn-group" dropdown>
         <button class="btn dropdown-toggle" dropdownToggle id="button-basic" type="button"
-                aria-controls="dropdown-basic">
-          {{ name }} <img alt="" class="profile-picture" src="assets/janette-doe.png"> <span class="caret"></span>
+          aria-controls="dropdown-basic">
+          {{ name }} <img alt="" class="profile-picture" src="assets/janette-doe.png"> <span
```

Comment:
Why the empty alt attribute?

Reply:
It's for accessibility purposes and considered good practice, see https://davidwalsh.name/accessibility-tip-empty-alt-attributes

Reply:
But then why not actually set the alt attribute to something sensible instead of being empty?  I guess the real question is do we want to support these accessibility features.

Reply:
Because in this case the image is already being described by the neighboring text (the user's name) and it wouldn't add value to tell the user their name and the fact that an image of them is next to their name.

For the second question, yes, I would definitely argue in favor of supporting accessibility features, as you can read in my 7 factors framework plan, in which I also address accessibility.

Reply:
I agree that we should support the accessibility features, Google prefers websites that are accessible to everyone.